### ISSUE TITLE
Prot paladin sim not working

### ISSUE THREAD

**Issue by Drulludanni** [wowsims/wotlk#3439]
**Describe the bug**
There is something very weird going on with the prot paladin sim, sometimes TMI is 0, dps is sometimes negative the numbers definitely cant be trusted as is.

**To Reproduce**
[negative dps build](https://wowsims.github.io/wotlk/protection_paladin/#eJztVE9IFGEU3/dm22aGwnHZw7TkMs0lCza+nWWhgvjGLnVck0CigwZ5KcKEDm6XJRRUogIjcA8p6qGLkiftEPknA6Mls9CQBNct3VJkWxSEOvRmprXStozs1g/eN9/7vfd+85vhm5EPiaggw8NQBbUQB7gFMIsrwj2A+wDjlABkALKAa4BNCMd1ERRQQUMGUaikmSaADgDq70B4bbXBUVTA6/JPeK6BJ3qxuv58Hc24/WmQi5TWz6h+KNE93SM49ABlSelJoy7M03aXksqgej2gC0k7u0uda7Iu9FPmVfpzlJXoEtWo2eKKlDRxLQHdk8wTcRq5TcR6x6sc2iP9DuFVEm9RXXZkHCVqekIyd/baMknbUnveklt5lLLW0QVrXXxu2fo0j+pHX95yK+k1BxyTbqVnBXVRdLPdUaFS6MRVlAfR9R9/BS/704mYSMcNVdTgKYTloAYHbHqwzvRrMooeCpFCokAKgWInBVDs8KMqtECUFUSQRRiLhMIRg7FQxAgboTAhZG2CEWaEmcFCxqapOfCJM5Oo9L1DtWEItcwEli5MYAYwC+4ESMU1Nib5MLoGAKbBJ5220P7QLHYeCk3Nk0CX3ilIdh6vyVd85p5Em4UxfsRhxvm+vJ6Rw9qGGxiTyi5dObe/uq76glahigf/8bEsbTa3QyY+9vg3OhW/qs9OL9eXbYePn8F+1VWNS2YbJJ9ZeMO/3muNn/ruw8/yqa7aY/JUmtNCuxzfqNRtlbsGN/GF4PSP85OpmK/3xHsetvGSj6yWnr05PMqvEpuKdXP74psvqLtUPhBoXOxbrzt6Ke7oz3Kn/oJfto3P8MZFi0hv2edGWGrlA71bnvcyx4lsbnXiTIFfRZb/eP2GL7OcAIQ=)

![image](https://github.com/wowsims/wotlk/assets/9061317/0f6153ff-6056-48b9-805d-9643530b52ed)

**Expected behavior**
dps that makes sense

**Desktop (please complete the following information):**
Brave browser windows 10

**Comment by jarveson** [comment]
negative dps is expected for tank sims on anubarak, as the boss healing is merged with dps listing to show impact of nr swaps
